mochitest.sh

```diff
@@ -15,9 +15,17 @@ cd /activity-stream && npm install . && npm run buildmc
 cd /mozilla-central && ./mach build \
   && ./mach test browser_parsable_css \
   && ./mach lint -l codespell browser/components/newtab \
-  && ./mach test browser/components/newtab/test/browser --headless \
-  && ./mach test browser/components/newtab/test/xpcshell \
-  && ./mach test browser/components/preferences/in-content/tests/browser_hometab_restore_defaults.js --headless \
-  && ./mach test browser/components/preferences/in-content/tests/browser_newtab_menu.js --headless \
-  && ./mach test browser/components/enterprisepolicies/tests/browser/browser_policy_set_homepage.js --headless \
-  && ./mach test browser/components/preferences/in-content/tests/browser_search_subdialogs_within_preferences_1.js --headless
+  && ./mach test browser/components/newtab/test/browser \
+  browser/components/newtab/test/xpcshell \
+  browser/components/preferences/in-content/tests/browser_hometab_restore_defaults.js \
+  browser/components/preferences/in-content/tests/browser_newtab_menu.js \
+  browser/components/enterprisepolicies/tests/browser/browser_policy_set_homepage.js \
+  browser/components/preferences/in-content/tests/browser_search_subdialogs_within_preferences_1.js --log-tbpl test_run_log
```

Comment:
nit: move the `--log-tbpl` to the first line `./mach test --log-tbpl` and we can chain the individual test/paths below it to not hide this log option